.github/workflows/phpunit.yml

```diff
@@ -39,8 +39,9 @@ jobs:
         run: vendor/bin/phpunit -v --coverage-text --coverage-clover build/logs/clover.xml
 
       - name: Upload coverage results to Coveralls
+        if: ${{ matrix.dependency-version == 'prefer-lowest' }}
         env:
           COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          composer global require php-coveralls/php-coveralls
+          composer global require php-coveralls/php-coveralls guzzlehttp/guzzle:^7.0 --prefer-lowest
```

Comment:
Coveralls does not support Symfony 6 right now, but that should not prevent us from making this package compatible.